tools/PI/DevHome.PI/PIApp.xaml

```diff
@@ -2,7 +2,9 @@
 <Application
     x:Class="DevHome.PI.App"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:behaviors="using:DevHome.Common.Behaviors">

```

Comment:
Curious, why do we need these?

Reply:
Leftover from when I had a more complex breadcrumb style. Removed in 2nd commit.